src/SwedbankPay/Api/Service/Request.php

```diff
@@ -211,6 +274,72 @@ public function send()
             throw new ClientException('PAYEE_ID not configured in INI file or environment variable.');
         }
 
+        // Retrieve parameters of operation
+        $paymentId = $this->getPaymentId();
+        $paymentOrderId = $this->getPaymentOrderId();
+        $operationRel = $this->getOperationRel();
+        if (($paymentId || $paymentOrderId) && $operationRel) {
+            // Fetch payment info
+            $responseBody = $this->getClient()->request(
+                'GET',
+                $paymentId ? $paymentId : $paymentOrderId,
+                $this->getRequestResource()
+            )->getResponseBody();
+
+            /** @var ResponseInterface $response */
+            $response = new Response($this, $responseBody);
+
+            // Operations
+            if (in_array($this->getOperationRel(),
+                [
+                    'sales',
+                    'transactions',
+                    'verifications',
+                    'authorizations',
+                    'captures',
+                    'cancellations',
+                    'reversals',
+                    'urls',
+                    'payeeInfo',
+                    'metadata',
+                    'payer',
+                    'payments',
+                    'current_payment'
+                ]
+            ) ) {
+                // Configure request
+                $responseData = $response->getResponseData();
+                if (isset($responseData['payment'][$this->getOperationRel()])) {
+                    $this->setRequestEndpoint($responseData['payment'][$this->getOperationRel()]['id']);
+                } elseif (isset($responseData['payment_order'][$this->getOperationRel()])) {
+                    $this->setRequestEndpoint($responseData['payment_order'][$this->getOperationRel()]['id']);
+                } else {
```

Comment:
Codacy found an issue: [The method send uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6123563)